### ISSUE TITLE
Search messages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#55]
```
Feature request: The ability to search messages.

Example: search:car, search:Bob:liquorice
```

Original issue reported on code.google.com by `Eckankar` on 20 Jan 2011 at 10:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 12:54
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 4:29

**Comment by GoogleCodeExporter** [comment]
```
new command created:

findsms:#message#
findsms:#contact#:#message#

Alias "fs" works too
```

Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 6:49
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.8
```

Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 10:57
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
